src/main/java/org/betonquest/betonquest/compatibility/protocollib/api/GlowAPI.java

```diff
@@ -0,0 +1,238 @@
+     * Creating and Sending Glow Packet in async (background)
+     *
+     * @param entity  Target entity that will get glow
+     * @param glowing true if entity need to be glowing
+     * @param player  Target Player that can see the glow
+     * @return void
+     */
+    private CompletableFuture<Void> sendGlowPacketAsync(final Entity entity, final boolean glowing, final Player player) {
+        return CompletableFuture.runAsync(() -> {
+            final WrappedDataWatcher.WrappedDataWatcherObject dataWatcherObject = new WrappedDataWatcher.WrappedDataWatcherObject(0, WrappedDataWatcher.Registry.get(Byte.class));
+            final PacketContainer packetContainer = new PacketContainer(PacketType.Play.Server.ENTITY_METADATA);
+
+            final int entityId = entity.getEntityId();
+
+            final WrappedDataWatcher dataWatcher = WrappedDataWatcher.getEntityWatcher(entity);
+            final List<WrappedWatchableObject> dataWatcherObjects = dataWatcher.getWatchableObjects();
+
+            byte entityByte = 0x00;
+            if (!dataWatcherObjects.isEmpty()) {
+                entityByte = (byte) dataWatcherObjects.get(0).getValue();
+            }
+            if (glowing) {
+                entityByte = (byte) (entityByte | 0x40);
+            } else {
+                entityByte = (byte) (entityByte & ~0x40);
+                final Collection<Entity> entities = Collections.singletonList(entity);
+                sendTeamPacketAsync(entities, null, Mode.TEAM_REMOVED, player);
+            }
+
+            final WrappedWatchableObject wrappedMetadata = new WrappedWatchableObject(dataWatcherObject, entityByte);
+            final List<WrappedWatchableObject> metadata = Collections.singletonList(wrappedMetadata);
+
+            packetContainer.getIntegers().write(0, entityId);
+            packetContainer.getWatchableCollectionModifier().write(0, metadata);
+
+            sendPacket(player, packetContainer);
+        });
+    }
+
+    /**
+     * Sending a PacketContainer.
+     *
+     * @param player          Send PacketContainer to Player
+     * @param packetContainer PacketContainer that will get send
+     */
+    private static void sendPacket(final Player player, final PacketContainer packetContainer) {
+        try {
+            ProtocolLibrary.getProtocolManager().sendServerPacket(player, packetContainer);
+        } catch (final InvocationTargetException e) {
+            LOG.warn("Error while sending the Packet", e);
+        }
+    }
+
+    /**
+     * Creating and Sending Team Packet in async (background)
+     *
+     * @param entities   List of entities that will be on team
+     * @param color      Color of the team
+     * @param packetMode Mode of the team
+     * @param player     Player that have the team
+     * @return void
+     */
+    public static CompletableFuture<Void> sendTeamPacketAsync(final Collection<? extends Entity> entities, final ChatColor color, final Mode packetMode, final Player player) {
```

Comment:
In this class a lot of methods to have the void return type, while actually returning a CompletableFuture. this is simply a incorrect javadoc

while this method has a return, while it is never used, but i think this is not an issue